### PR TITLE
Update PHP SDK docs for v7

### LINKED ISSUE #419: JWTVerifier alternatives
Document says that JWTVerifier is used to decod token,  but the current version has no JWTVerifier. How can I decode token?

https://auth0.com/docs/libraries/auth0-php/jwt-validation

### PR DESCRIPTION
**Pull request by joshcanhelp**
The PHP SDK just had a [major release](https://github.com/auth0/auth0-PHP/releases/tag/7.0.0) and the documentation needed an update based on breaking changes.

Closes https://github.com/auth0/auth0-PHP/issues/419

### CODE REVIEW

File: articles/libraries/auth0-php/authentication-api.md [hunk 1]
```diff
@@ -124,32 +125,38 @@ $auth0_api = new Authentication(
...
 1. After the state validation, we instantiate an instance of the `Authentication` class with the Domain, Client ID, and the Client Secret, which is used with the `code` to obtain an access token.
 1. We attempt an exchange with the `$auth0_api->code_exchange()` call, making sure to pass in the same redirect URI we used to request the `code`.
-1. If this succeeds, we know the exchange was successful, and we have an access token. We call the `/userinfo` endpoint with this token, which will return an object containing the data we requested in the `scopes` parameter on the authorize URL.
+1. If this succeeds, we know the exchange was successful, and we have an <dfn data-key="id-token">ID Token</dfn> and an <dfn data-key="access-token">Access Token</dfn>. We validate the <dfn data-key="id-token">ID Token</dfn> and use the claims for the user identity.
 1. If this last step succeeds, we store the user and redirect back to our sensitive data.
```
Comment (stevehobbsdev) [inline articles/libraries/auth0-php/authentication-api.md:160]: All of these items are prefixed with `1.`.. assuming they should be sequential?

File: articles/libraries/auth0-php/basic-use.md [hunk 2]
```diff
@@ -18,15 +18,14 @@ The examples below assume that you followed the steps in the [Installation and G
 
 ### Login
 
-The basic login process in Auth0-PHP uses an [Authentication Code grant](/api-auth/tutorials/authorization-code-grant) combined with Auth0's <dfn data-key="universal-login">Universal Login</dfn> page. In short, that process is:
+The default login process in Auth0-PHP uses an [Authentication Code grant](/api-auth/tutorials/authorization-code-grant) combined with Auth0's <dfn data-key="universal-login">Universal Login</dfn> page. In short, that process is:
```
Comment (stevehobbsdev) [inline articles/libraries/auth0-php/basic-use.md]: > combined with Auth0's <dfn data-key="universal-login">Universal Login</dfn> page

"page" is lowercase here but uppercase in other places where you have this kind of reference to the ULP - ideally should be consistent.

File: articles/libraries/auth0-php/jwt-validation.md [hunk 3]
```diff
@@ -29,45 +31,40 @@ if (empty($_GET['token_alg']) || ! in_array($_GET['token_alg'], [ 'HS256', 'RS25
...
 
-- **cache**: Receives an instance of `Auth0\SDK\Helpers\Cache\CacheHandler` (Supported `FileSystemCacheHandler` and `NoCacheHandler`). Defaults to `NoCacheHandler` (RS256 only).
-- **guzzle_options**: Configuration propagated to Guzzle when fetching the JWKs (RS256 only). These options are [documented here](http://docs.guzzlephp.org/en/stable/request-options.html).
-- **secret\_base64\_encoded**: When `true`, it will decode the secret used to verify the token signature. This is only used for HS256 tokens and defaults to `true`. Your Application settings will say whether the Client Secret provided is encoded or not.
+* `nonce` to check the nonce contained in the token (recommended for OIDC-compliant tokens).
+* `max_age` to check the auth_time of the token.
+* `leeway` clock tolerance in seconds for the current check only (60 seconds default).
 
-### Read more
+## Read more
```
Comment (stevehobbsdev) [inline articles/libraries/auth0-php/jwt-validation.md]: Other sub-heads in this doc use Title Case, would be good to be consistent.

```suggestion
## Read More
```

**Review by stevehobbsdev — CHANGES_REQUESTED** [review]
LGTM, just left a couple of things.

File: articles/libraries/auth0-php/authentication-api.md (hunk 1 above)
Comment (joshcanhelp) [reply articles/libraries/auth0-php/authentication-api.md:160]: They are output sequentially when rendered.

File: articles/libraries/auth0-php/basic-use.md (hunk 2 above)
Comment (joshcanhelp) [reply articles/libraries/auth0-php/basic-use.md]:
```suggestion
The default login process in Auth0-PHP uses an [Authentication Code grant](/api-auth/tutorials/authorization-code-grant) combined with Auth0's <dfn data-key="universal-login">Universal Login</dfn> Page. In short, that process is:
```

**Review by stevehobbsdev — APPROVED** [review]

**Pull request merged**